lib/aff4_test.py

```diff
@@ -471,6 +473,9 @@ def ExerciseAFF4ImageBase(self, classname):
     self.assertEqual(len(data), 300)
     self.assertTrue("XXXHello WorldXXX" in data)
     self.assertTrue("XXXYYY" in data)
+    fd.seek(0)
```

Comment:
add a comment about what you're testing and also test the non-failure case that you get 1byte back from a read.